packages/frontend/src/exports/components.ts

```diff
@@ -14,6 +14,7 @@ export { default as ChooseConnectionSubstep } from '../components/ChooseConnecti
 export { default as ConditionalIconButton } from '../components/ConditionalIconButton'
 export { default as Container } from '../components/Container'
 export { default as ControlledAutocomplete } from '../components/ControlledAutocomplete'
+export { default as DebouncedSearchInput } from '../components/DebouncedSearchInput'
```

Comment:
actually do we need to add new ones to this list?

Reply:
not really. but in case admin panel need